packages/grid/x-data-grid-pro/src/components/DataGridProVirtualScroller.tsx

```diff
@@ -274,8 +328,75 @@ const DataGridProVirtualScroller = React.forwardRef<
 
   const detailPanels = getDetailPanels();
 
+  const topPinnedRows = getRows({ renderContext, rows: topPinnedRowsData });
+
+  const pinnedRowsHeight = calculatePinnedRowsHeight(apiRef);
+
+  const mainRows = getRows({
+    renderContext,
+    rowIndexOffset: topPinnedRowsData.length,
+  });
+
+  const bottomPinnedRows = getRows({
+    renderContext,
+    rows: bottomPinnedRowsData,
+    rowIndexOffset: topPinnedRowsData.length + (mainRows ? mainRows.length : 0),
+  });
+
+  const contentProps = getContentProps();
+
+  const pinnedColumnsStyle = { minHeight: contentProps.style.minHeight };
+
+  if (contentProps.style.minHeight && contentProps.style.minHeight === '100%') {
+    contentProps.style.minHeight = `calc(100% - ${pinnedRowsHeight.top}px - ${pinnedRowsHeight.bottom}px)`;
+  }
+
   return (
     <GridVirtualScroller {...getRootProps(other)}>
+      {topPinnedRowsData.length > 0 ? (
+        <VirtualScrollerPinnedRows
+          className={classes.topPinnedRows}
+          ownerState={{ position: 'top' }}
+          style={{ width: contentProps.style.width, height: pinnedRowsHeight.top }}
```

Comment:
These intermediary DIVs pollute the accessibility tree with "generic".

Before:

<img width="300" src="https://user-images.githubusercontent.com/42154031/181017882-01b5d97b-5dc2-4421-ba19-f8cfb20f85aa.png" alt="image" style="max-width: 100%;">

After: 

<img width="300" src="https://user-images.githubusercontent.com/42154031/181018082-9d3931aa-f194-447e-86bb-0d1749349c74.png" alt="image" style="max-width: 100%;">


```suggestion
          style={{ width: contentProps.style.width, height: pinnedRowsHeight.top }}
          role="rowgroup"
```

Same for the virtual scroller of the bottom pinned rows.

We can later add `presentation` and `rowgroup` to other elements too.